mirror_list.txt

```diff
@@ -1,5 +1,3 @@
-golang:latest
-node:lts-slim
-node:lts
-composer:2.1.11
-python:slim
+composer:2.4
+java-quarkus-image:latest
```

Comment:
Wait, these last two images don't exist

Reply:
`java-quarkus-image:latest` 
`java-springboot-image:latest`

These are the templated outerloop images, we don't need to mirror them.